apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePagesTable.tsx

```diff
@@ -91,6 +91,9 @@ export const ManagePagesTable = ({ summaries, currentPage, pageSize, totalElemen
                 case Column.Status:
                     setSortBy('status');
                     break;
+                case Column.RelatedConditions:
+                    setSortBy('conditions');
+                    break;
```

Comment:
Sorting by conditions is not yet supported on the API. We need to have a discussion with the design team to ensure this is something we want to do. 

If a page is associated with condition `aaa` and condition `zzz`. No matter the direction of the sort, this page will be first. I have concerns over the usefulness of sorting on a multi-valued field.

If you have time, you could connect with Chang or Jared and verify the expected behavior

Reply:
Yeah, i can remove this if we decide not to allow the sorting. for now I have the config set to false so that the sorting arrow will not appear.

Reply:
I will follow up with designers